udata_front/theme/gouvfr/assets/js/components/SchemaSelect/SchemaSelect.vue

```diff
@@ -41,8 +41,8 @@ export default defineComponent({
   setup(props, {emit}) {
     /** @type {Promise<Array<import("../../types").MultiSelectOption>>} */
     const initialOptions = getCatalog().then(catalog => catalog.map(schema => ({
-      label: schema.name,
-      value: schema.name,
+      label: schema.id,
+      value: schema.id,
```

Comment:
I think this change broke the frontend… We should keep `.name` here. 
![image](https://github.com/etalab/udata-front/assets/1770543/eca2c9d0-66b5-4938-9495-c995a4498866)

Reply:
The PR is not working right now and is waiting for https://github.com/etalab/udata-front/pull/378 to be merged (for the [api file](https://github.com/etalab/udata-front/pull/378/files#diff-e9f6af4ac65a05931acf1a1beb71e93ce4ddbfa3e920bf2d7af0514e738ec127) in datagouv-components).

Then, a new version of the package is necessary to allow schema api changes to be reflected in udata-front.